src/utils/index.ts

```diff
@@ -42,3 +42,58 @@ export function scaleGraph(ogma: Ogma, scale: number) {
 
 export const stringtoBase64 = (str: string) =>
   btoa(unescape(encodeURIComponent(str)));
+
+export const stopPropagation = (e: Event) => e.stopPropagation();
+
+export type Bounds = [number, number, number, number];
+
+export function mergeBounds(b0: Bounds, b1?: Bounds): Bounds {
+  if (!b1) return b0;
+  return [
+    Math.min(b0[0], b1[0]),
+    Math.min(b0[1], b1[1]),
+    Math.max(b0[2], b1[2]),
+    Math.max(b0[3], b1[3]),
+  ];
+}
+
+export function getBoundingBox(ogma: Ogma, texts: boolean): Bounds {
```

Comment:
I thought I made someting like that in ogma: with https://doc.linkurious.com/ogma/4.4/api.html#NodeList-getBoundingBox and include texts. Is it buggy ?

Reply:
yes something stopped me there, maybe it didn't take the texts that were not rendered, into account

Reply:
![Screenshot 2023-05-23 at 12 02 36](https://github.com/Linkurious/lke-plugin-image-export/assets/26884/4cb8458b-113e-43fa-b3ef-663f02613a66)
I don't know why